### PR TITLE
Added verify client app approval endpoint

### LINKED ISSUE #121: Verification endpoint to client apps and add pin code to employee

### PR DESCRIPTION
**Pull request by lauriLukkarinen**
- also added PIN code for employee

Resolves #121

### CODE REVIEW

File: services/user-management-services.yaml [hunk 1]
```diff
@@ -1158,6 +1158,37 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/clientApps/verifyApproval:
```
Comment (anttileppa) [inline services/user-management-services.yaml:1161]: I would prefer it to be top-level endpoint, e.g.

```suggestion
  /v1/verifyClientApp:
```

File: services/user-management-services.yaml [hunk 2]
```diff
@@ -1158,6 +1158,37 @@ paths:
...
+    post:
+      operationId: verifyClientAppApproval
```
Comment (anttileppa) [inline services/user-management-services.yaml:1163]:
```suggestion
      operationId: verifyClientApp
```

File: services/user-management-services.yaml [hunk 3]
```diff
@@ -1158,6 +1158,37 @@ paths:
...
+      summary: Verifies client app approval.
```
Comment (anttileppa) [inline services/user-management-services.yaml:1164]:
```suggestion
      summary: Verifies that the client app has been approved.
```

File: services/user-management-services.yaml [hunk 4]
```diff
@@ -1158,6 +1158,37 @@ paths:
...
+      tags:
+        - ClientApps
+        - SpecAuth
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: string
+              description: Device ID
```
Comment (anttileppa) [inline services/user-management-services.yaml:1177]: plain string is not json, could this be moved into separate ClientAppVerifyRequest -object for ease of use?

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: services/user-management-services.yaml (hunk 2 above)
Comment (lauriLukkarinen) [reply services/user-management-services.yaml]: Modified.

File: services/user-management-services.yaml (hunk 3 above)
Comment (lauriLukkarinen) [reply services/user-management-services.yaml]: Updated.

File: services/user-management-services.yaml (hunk 4 above)
Comment (lauriLukkarinen) [reply services/user-management-services.yaml]: Extracted to VerifyClientAppRequest schema.

**Review by anttileppa — APPROVED** [review]

**Pull request merged**